nntrainer/layers/pooling2d_layer.cpp

```diff
@@ -40,25 +41,55 @@ int Pooling2DLayer::initialize(Manager &manager) {
     ml_logw("Warning: the length of previous layer dimension is one");
   }
 
+  if (pooling_type == PoolingType::global_max ||
+      pooling_type == PoolingType::global_average) {
+    if (pool_size[0] != in_dim.height() || pool_size[1] != in_dim.width()) {
+      ml_logw(
+        "[Pooling2D] global_max, global_average does not except pool size");
+      pool_size[0] = in_dim.height();
+      pool_size[1] = in_dim.width();
+    }
+
+    if (padding[0] != 0 || padding[1] != 0) {
```

Comment:
Is this something that our implementation does not support or it is something that is just not possible?

Reply:
Global pooling has no notion of padding, stride, pool_size